blog/2023/06/PSA-dont-use-threadid.md

```diff
@@ -0,0 +1,175 @@
++++
+mintoclevel = 2
+maxtoclevel = 3
+title = "PSA: Thread-local state is no longer recommended; Common misconceptions about threadid() and nthreads()"
```

Comment:
This is rendering pretty long and hard to read. Do we have a way to do sub-titles or something?